### PR TITLE
Ls 15 to MAIN PR

### PR DESCRIPTION
**Pull request by special-ded**
Ls 15 to MAIN PR

### CODE REVIEW

File: store/angular.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "src/scss/styles.scss"
```
Comment (philippbulanin) [inline store/angular.json:33]: Do not replace global `styles.scss` file

File: store/src/app/app-routing.module.ts [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: '',
+    redirectTo: '/products',
+    pathMatch: 'full'
+  },
+  {
+    path: 'products',
+    loadChildren: () => import('./products-routing/products-routing.module').then((m) => ProductsRoutingModule)
```
Comment (philippbulanin) [inline store/src/app/app-routing.module.ts:13]: Should be ` .then(m => m.ProductsModule)`

You loading main `ProductsModule`, which includes `ProductsRoutingModule`

File: store/src/app/app.module.ts [hunk 3]
```diff
@@ -1,16 +1,21 @@
...
 
 @NgModule({
   declarations: [
     AppComponent
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+    HttpClientModule,
```
Comment (philippbulanin) [inline store/src/app/app.module.ts:17]: Do you need `HttpClientModule`? Why?

File: store/src/app/app.module.ts [hunk 4]
```diff
@@ -1,16 +1,21 @@
...
+    ProductsModule
```
Comment (philippbulanin) [inline store/src/app/app.module.ts:18]: You do not need to include `ProductsModule` here. It is inluding by lazyLoading in routing

File: store/src/app/products/color-price.directive.ts [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+  @Input() public price: number = 0;
+
+  constructor(private el: ElementRef) { }
+
+  ngOnInit(): void {
+    this.setPriceColor(this.price)
+  }
+
+  setPriceColor(price: number) {
```
Comment (philippbulanin) [inline store/src/app/products/color-price.directive.ts:17]: Add return type

**Review by philippbulanin — CHANGES_REQUESTED** [review]

File: store/angular.json (hunk 1 above)
Comment (special-ded) [reply store/angular.json:33]: Done

File: store/src/app/app.module.ts (hunk 3 above)
Comment (special-ded) [reply store/src/app/app.module.ts:17]: Deleted

File: store/src/app/app.module.ts (hunk 4 above)
Comment (special-ded) [reply store/src/app/app.module.ts]: Done

File: store/src/app/products/color-price.directive.ts (hunk 5 above)
Comment (special-ded) [reply store/src/app/products/color-price.directive.ts:17]: Done

**Pull request merged**